### DISCUSSION TITLE
Error: Cannot read properties of undefined (reading 'some')

### DISCUSSION THREAD

**Discussion by WEARNXT** [AntlerVC/firetable#1037] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**

Added a 'created at' column to at sub-collection, and now the project fails to load.

**Comment by BeeBombshell** [comment]
Hey @WEARNXT, I tried to recreate the issue you're facing but, my project loads just fine. I'm not sure if this is causing the error. Could you provide some more information or screenshots so that we can help you out?

**Reply by WEARNXT** [reply]
Hey, The error I'm receiving now is:

Oops, something went wrong
This issue has been reported to the Rowy team to be fixed.

ROWY TRACING ID: ROWYERR-F917-CEEA-490C-78540106AD05


Please let me know if there is a way to track this issue.

**Comment by bofeiw** [comment, marked as answer]
I have created a PR in an attempt to fix this issue: https://github.com/rowyio/rowy/pull/1060

**Comment by WEARNXT** [comment]
Hey Bobby, I'm still facing the same issue, and also unable to track the
progress being made towards this via the rowy tracking system:
ROWY TRACING ID: ROWYERR-2715-8DBD-7030-8E1BF357857D
I look forward to hear more about this,

Thank you

**Reply by bofeiw** [reply]
The fix is not merged to to Rowy.io yet, we will release the fix soon and hopefully it will fix the issue.

**Comment by WEARNXT** [comment]
Hey Bobby,

That sounds great, thank you so much.
I look forward to it.

Best Regards,

Zeeshan Ahmed

**Reply by bofeiw** [reply]
Sorry for the late reply. The attempted fix is deployed to rowy.app. Are you still experiencing the issue?

**Comment by WEARNXT** [comment]
Hi,

The problem is solved for me,
Thank you so much.

Best Regards,
Zeeshan Ahmed
